services/api/README.md

```diff
@@ -0,0 +1,32 @@
+<p align="center"><img
+src="https://raw.githubusercontent.com/amazeeio/lagoon/master/docs/images/lagoon-logo.png"
+alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut"
+width="40%"></p>
+
+This service is part of amazee.io Lagoon, a Docker build and deploy system for
+OpenShift & Kubernetes. Please reference our [documentation] for detailed
+information on using, developing, and administering Lagoon.
+
+# API (`api`)
+
+The main GraphQL API for Lagoon. Uses the [Apollo server library](https://www.apollographql.com/docs/apollo-server/v1/).
+
+## Technology
+
+* Node
+* GraphQL
+* Message Queue
+
+## Related Services
+
+* API [***dependency***]
```

Comment:
Would it make sense to link to these? Both internal and external services - I like to err on the side of making sure folks know exactly what we're talking about.

Reply:
I didn't link as I was assuming the "documentation" link in the opening paragraph would suffice  to avoid a lot of re-work in case structure changes. If we link, do we link directly to the other README files in github or to pages in /docs?

There are no "external" services in any of these AFAIK.